docs/app/Examples/views/Card/Content/CardExampleLinkCard.js

```diff
@@ -2,12 +2,20 @@ import React from 'react'
 import { Card } from 'semantic-ui-react'
 
 const CardExampleLinkCard = () => (
-  <Card
-    href='#link'
-    header='Elliot Baker'
-    meta='Friend'
-    description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
-  />
+  <Card.Group>
+    <Card
+      href='#link'
+      header='Elliot Baker'
+      meta='Friend'
+      description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
+    />
+    <Card
+      link
+      header='Rick Sanchez'
+      meta='Scientist'
+      description='Rick is a genius scientist whose alcoholism and reckless, nihilistic behavior are a source of concern for his family'
+    />
+  </Card.Group>
```

Comment:
Okay, but in the actual [docs](http://semantic-ui.com/views/card#link-card), they're next to each other. Since they're showing the same property functionality, shouldn't they be together?